changelog.txt

```diff
@@ -75,7 +111,7 @@
 	* Refine display of post status in the lesson block [#3680](https://github.com/Automattic/sensei/pull/3680)
 	* Remove inserter from module block and adjust how empty lesson block is added [#3684](https://github.com/Automattic/sensei/pull/3684)
 	* Remove module animation setting and replace with collapsible modules setting [#3675](https://github.com/Automattic/sensei/pull/3675)
-	* Set a default value for `outlineClassName` in the module block [#3806](https://github.com/Automattic/sensei/pull/3806) 
+	* Set a default value for `outlineClassName` in the module block [#3806](https://github.com/Automattic/sensei/pull/3806
```

Comment:
This seems like a mistake

Reply:
Thanks for find the oopsie. https://github.com/Automattic/sensei/pull/3956/commits/2a41b17ccb695b42b156832ad23e43a74583b7b8